pages/index.tsx

```diff
@@ -98,6 +99,18 @@ const Icebreaker: NextPage<Props> = ({
           handleCopyUrlClick={handleCopyUrlClick}
         />
       </Card>
+      <div
+        className="group mx-4 mt-auto mb-4 inline-flex flex-col items-center justify-center gap-2 text-center text-white"
```

Comment:
-1 `group` isn't needed here; use case for `group` is valid when you need hover state for more complex HTML structure - https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state